### DISCUSSION TITLE
How i can make custom map adaption?

### DISCUSSION THREAD

**Discussion by GreenGroove46** [sultim-t/xash-rt#101] [Q&A] [answered]
I want to make rt compability for custom mappack, but how?

**Comment by sultim-t** [comment, marked as answer]
Soon, there will be the instructions!

**Reply by GreenGroove46** [reply]
Thanks!

**Reply by THG327** [reply]
> Soon, there will be the instructions!

Is there an update to this?

**Reply by sultim-t** [reply]
A draft https://github.com/sultim-t/xash-rt/discussions/150
